src/main/java/com/minecolonies/entity/ai/util/ChatSpamFilter.java

```diff
@@ -17,37 +17,36 @@
      * 20 Ticks per second.
      * 30 Seconds.
      */
-    protected static final int BASE_TIMEOUT  = 30 * 20;
+    static final int BASE_TIMEOUT = 30 * 20;
+
     /**
      * The maximum delay to wait
      * <p>
      * 20 Ticks per second.
      * 60 Seconds per Minute.
      * 10 Minutes.
      */
-    protected static final int MAX_TIMEOUT   = 20 * 60 * 10;
-    /**
-     * The number to multiply timeout with every time
-     * <p>
-     * BASE_TIMEOUT * 2^times
-     */
-    private static final   int POWER_TIMEOUT = 2;
+    static final int MAX_TIMEOUT = 20 * 60 * 10;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "MAX_TIMEOUT". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)